1.4/Patches/Mods/Viles Materials.xml

```diff
@@ -101,8 +101,8 @@
 				<xpath>Defs/ThingDef[defName="Plastic"]</xpath>
 				<value>
 					<li Class="Celsius.StuffThermalProperties">
-						<volumetricHeatCapacity>700</volumetricHeatCapacity>
-						<insulationFactor>0.8</insulationFactor>
+						<volumetricHeatCapacity>1550</volumetricHeatCapacity>
```

Comment:
I dont think plastic should change. It doesnt insulate, and it ahs low heat capacity, meaning it will heat up and cool very fast,